src/plugins/layout/stateprovider.ts

```diff
@@ -129,3 +127,34 @@ export function createGenericLayoutState(node: Node, tag: string): ILayoutState
   };
   return layoutState;
 }
+
+export function shouldIgnoreNode(node: Node, shadowDom: ShadowDom): boolean {
+  let shadowNode = shadowDom.getShadowNode(getNodeIndex(node));
+  let ignore = false;
+  switch (node.nodeType) {
+    case Node.ELEMENT_NODE:
+      let tagName = (node as Element).tagName;
+      if (tagName === "SCRIPT" || tagName === "META") {
```

Comment:
>"META" [](start = 46, length = 6)

a bit of a nit, but for these string constants that get used multiple times, you'll get better crunching if you move them to a variable